Resources/Views/index.leaf

```diff
@@ -28,38 +34,22 @@
 
     <link rel="canonical" href="#(canonicalUrlString)" />
 
-    <link rel="stylesheet" type="text/css" href="external/select2-4.0.13/select2.min.css">
+    <link href='css/fonts/ProximaNova-Regular.woff2' rel='stylesheet' type='text/css'>
+    <link href='css/fonts/ProximaNova-Semibold.woff2' rel='stylesheet' type='text/css'>
```

Comment:
You also need to add the other font versions, besides `woff2` there's also `woff` and `opentype`, as it is right now font doesn't load on Firefox.

Reply:
You're right! Forgot to add the other two types.
And you're also right about the preload.
Thank you, Diego! Spot on as always!